addons/hr_timesheet/views/project_sharing_views.xml

```diff
@@ -14,28 +15,24 @@
                 <field name="remaining_hours" widget="timesheet_uom" sum="Remaining Hours" optional="hide" decoration-danger="progress &gt;= 100" decoration-warning="progress &gt;= 80 and progress &lt; 100"/>
                 <field name="progress" widget="progressbar" optional="hide"/>
             </xpath>
+            <xpath expr="//field[@name='date_deadline']" position="before" groups="hr_timesheet.group_hr_timesheet_user">
```

Comment:
The portal user never has this group and so the content in the xpath will no longer displayed in project sharing. 😬